views/drones/create-form.hbs

```diff
@@ -1 +1,19 @@
-<h2>Create a new drone</h2>
\ No newline at end of file
+<h2>Create a new drone</h2>
+<form action="/drones/create" method="post">
+   <div class="mb-3">
+        <label for="name">Name</label>
+        <input required maxlength="250" type="text" name="name">
```

Comment:
Great HTML validation. Try to add always the same [validation](https://mongoosejs.com/docs/validation.html) in the model schema.